templates/admin/search.php

```diff
@@ -53,6 +57,15 @@
 				  <input class="term" type="text" name="replace_pattern" value="<?php echo esc_attr( $replace ) ?>"/><br/>
 				</td>
 			</tr>
+			<tr>
```

Comment:
We would prefer having this off by default, enabled by setting the `PB_ENABLE_REGEX_SEARCHREPLACE` constant to `true`.